src/schemas/user.schema.js

```diff
@@ -2,15 +2,57 @@ var mongoose = require('mongoose');
 var Schema = mongoose.Schema;
 
 var userSchema = new Schema({
-  name: {
+  Id: {
```

Comment:
@melvinpurio why did you add Id property for userSchema? I think the mongodb add unique id always every insert of a document

Reply:
I did notice that when i tried to output a user using an ID. Does it mean, every row has the same format of ID?

Reply:
I did notice that when i tried to output a user using an ID. Does it mean, every row has the same format of ID?

Reply:
I did notice that when i tried to output a user using an ID. Does it mean, every row has the same format of ID?

Reply:
I did notice that when i tried to output a user using an ID. Does it mean, every row has the same format of ID?

Reply:
I did notice that when i tried to output a user using an ID. Does it mean, every row has the same format of ID?

Reply:
yes they should have the same format. we should stick to use mongoose id, for now.

Reply:
Ok, got it